src/altinn-app-frontend/src/shared/resources/appLanguage/appLanguageReducer.ts

```diff
@@ -0,0 +1,69 @@
+import update from 'immutability-helper';
+import { Action, Reducer } from 'redux';
+import {
+  IFetchAppLanguagesFulfilled,
+  IFetchAppLanguagesRejected,
+  IUpdateAppLanguages,
+} from './fetch/fetchAppLanguagesActions';
+import * as ActionTypes from './fetch/fetchAppLanguagesActionTypes';
+
+import { IAltinnWindow } from 'src/types';
+import { IAppLanguage } from 'altinn-shared/types';
+const altinnWindow = window as Window as IAltinnWindow;
+const { app } = altinnWindow;
```

Comment:
Would it be possible to get the language key part from the URI instead? `app` could possibly not be defined in a unit-test for instance.